HTTPServerProject/HTTPServerProxy/Client.cs

```diff
@@ -0,0 +1,13 @@
+using System.Net.Sockets;
+
+namespace HTTPServerProxy.Client;
+
+public class ProxyClient
+{
+    TcpClient client = new TcpClient("127.0.0.1", 8000);
+
+    public NetworkStream GetStream()
```

Comment:
I'm not sure what purpose this class is serving anymore - it's just a getter. Now maybe if it TOLD the client to do something, I could see it.

Reply:
Agreed, I'm going to remove this class.